protected/humhub/modules/content/controllers/SearchContainerController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace humhub\modules\content\controllers;
+
+use humhub\modules\content\Module;
+use humhub\modules\space\controllers\BrowseController;
+use humhub\modules\user\widgets\UserPicker;
+use Yii;
+
+/**
+ * @property Module $module
+ */
+class SearchContainerController extends BrowseController
```

Comment:
Extending BrowseController is to make this controller simple and easy to maintain.
But:
- currently it is not used (it could potentially by using the `ContentContainerPicker` with the default search route), because only the Share Intend uses it, but it has a specific search controller that checks if we can create a content
- Theoretically, it should allow searching any Space or User, instead of returning "My Profile" and the searched Space), but it would be more complicated

Options:
- Remove it and force defining a route when using `ContentContainerPicker`ContentContainerPicker
- Add the possibility to search for Users too
- Create a common search controller for `ContentContainerPicker`, `UserPickerField` and `SpacePickerField`

Reply:
I agree. Done in commit https://github.com/humhub/humhub/pull/7455/commits/7c752677dd1f0d71432ea23d98766af4d2be134f